### ISSUE TITLE
10hitmovies

### ISSUE THREAD

**Issue by Rachitidk** [phisher98/cloudstream-extensions-phisher#234]
### Source name

10hitmovies

### Source link

https://10hitmovies.com/

### Source language

Hindi

### Other details

For MTV Hustle, tried other resources as well, but some contain wrong episode (mention on discord) or only one episode, some play bad quality even in 1080p
![1000111021](https://github.com/user-attachments/assets/b030f2d2-b2cb-4e52-8a69-bfa12f4b9770)
![1000111019](https://github.com/user-attachments/assets/99a61163-9a6b-4534-ba62-19fd1b3b92de)
, some shows remote error (created a GitHub issue, CSX).
This website contains all type of link
![1000111022](https://github.com/user-attachments/assets/a2f5309c-09bf-4cd3-91e7-d652f9c6b5da)
Or if there other sources, I am happy to use. Idk about this website, found while searching for hustle 4.

### Acknowledgements

- [X] I have searched the existing issues and this is a new ticket, **NOT** a duplicate or related to another open or closed issue.
- [X] I have written a title with source name.
- [x] I have checked that the extension does not already exist on the any Dev Repo or Request.
- [X] I have checked that the extension does not already exist by searching on CS and verified it.
- [X] I will fill out all of the requested information in this form.

**Comment by rockhero1234** [comment]
I recently added a pull request for cinedoze will be add soon on csx there you can watch hustle.

**Comment by rockhero1234** [comment]
I also added pull request for 10hitmovies on redowan repo  
Test it here included with 9kmovies https://pastebin.com/raw/54kbmF42
Tell me if working

**Comment by Rachitidk** [comment]
Ok wait

> I also added pull request for 10hitmovies on redowan repo Test it here :included with 9kmovies https://pastebin.com/raw/54kbmF42 Tell me if working

**Comment by Rachitidk** [comment]
![1000111031](https://github.com/user-attachments/assets/fa4de8fe-68b3-4cca-82c6-1db5dc36911d)


> I also added pull request for 10hitmovies on redowan repo Test it here :included with 9kmovies https://pastebin.com/raw/54kbmF42 Tell me if working

Which one

**Comment by rockhero1234** [comment]
9kmovies

**Comment by Rachitidk** [comment]
It's present in ss, if not there is no 9kmovies

**Comment by Rachitidk** [comment]
https://github.com/user-attachments/assets/e63f9d15-9571-454d-b2e4-1e39a5888990

**Comment by rockhero1234** [comment]
Turn on NSFW it contain 18+

**Comment by Rachitidk** [comment]
Tested: Working.
Some enhancement 
1) Same source repeated.
2) Same quality repeated.
3) Gdflix filix not working.
4) If possible can size of file be added (because I noticed for same quality streamtape and vidhidepro provide different size e.g. hustle s04e4 in cinedoze streamtape -900mb vid...- 2.2gb and the quality was way superior in vidhidepro)

https://github.com/user-attachments/assets/60c6887b-6cf7-43f4-87c8-6b28771d88c0

All problem are shown in record.
